src/main/java/frc/robot/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.elevator;
+
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import java.util.function.DoubleSupplier;
+import org.littletonrobotics.junction.Logger;
+
+/** Slanted cascading elevator */
+public class ElevatorSubsystem extends SubsystemBase {
+  // Constants
+  // TODO find real values
```

Comment:
We're fabbing. Does mech not have these yet?

Reply:
these constants did not exist when this was written and merged

Reply:
will do a pass over mechanisms to ensure all values are accurate